### ISSUE TITLE
Add standalone binary to released files

### ISSUE THREAD

**Issue by uberspot** [deepmind/kapitan#345]
After https://github.com/deepmind/kapitan/pull/323 we have a process to generate a standalone binary of kapitan and it's built in travis too.
The produced binary lives in dist/
It would be great to also include that to the files released by travis on each release.
i.e. in .travis.yml we already do that with CHANGELOG.md
```
deploy:
  - provider: releases
    api_key:
      secure: blabla=
    file: CHANGELOG.md
    prerelease: $PRERELEASE
    on:
      tags: true
      repo: deepmind/kapitan
```

**Comment by yoshi-1224** [comment]
@uberspot
I may be misunderstanding, but is this issue already addressed in #349?

**Comment by uberspot** [comment]
Yes it is. :) i just made an issue to track that work. I think the binary is now on the releases page on github and every build works fine on travis so i'd consider this issue resolved for now. :)
